app/src/main/java/com/mikhailovskii/weatherandroid/util/WeatherCalculations.kt

```diff
@@ -4,42 +4,34 @@ import android.annotation.SuppressLint
 import java.text.SimpleDateFormat
 import java.util.*
 
-fun getWindDirection(degree: Int): String {
-    when {
-        degree > 337.5 -> {
-            return "N"
-        }
-        degree > 292.5 -> {
-            return "NW"
-        }
-        degree > 247.5 -> {
-            return "W"
-        }
-        degree > 202.5 -> {
-            return "SW"
-        }
-        degree > 157.5 -> {
-            return "S"
-        }
-        degree > 122.5 -> {
-            return "SE"
-        }
-        degree > 67.5 -> {
-            return "E"
-        }
-        degree > 22.5 -> {
-            return "NE"
-        }
-        else -> {
-            return "N"
-        }
-    }
-}
-
 @SuppressLint("SimpleDateFormat")
 fun getDateFromSeconds(seconds: Long): String {
     val simpleDateFormat = SimpleDateFormat("E HH:mm")
     val date = Date(seconds * 1000)
 
     return simpleDateFormat.format(date)
-}
\ No newline at end of file
+}
+
+class WeatherUtils {
+
+    companion object {
+        private const val KELVIN_TO_CELSIUS_COEFFICIENT = 273
+
+        fun convertKelvinToCelsius(kelvin: Double): Double = kelvin - KELVIN_TO_CELSIUS_COEFFICIENT
+
+        fun getWindDirection(degree: Int): String {
+            return when {
+                degree > 337.5 -> "N"
+                degree > 292.5 -> "NW"
+                degree > 247.5 -> "W"
+                degree > 202.5 -> "SW"
+                degree > 157.5 -> "S"
+                degree > 122.5 -> "SE"
+                degree > 67.5 -> "E"
+                degree > 22.5 -> "NE"
+                else -> "N"
+            }
+        }
+
+    }
+}
```

Comment:
@SergeiMikhailovskii I would suggest to move out `getDateFromSeconds()` method into some other utility class (or file if you wish), since it doesn't relate to weather calculation IMO. Then you'll have only one class `WeatherUtils` in the file `WeatherCalculations`, so it would make sense to rename the file into `WeatherUtils`. 

Second point is `getDateFromSeconds` uses java SimpleDateFormat which is not recommended to use because of performance issues. There is a more efficient implementation for mobile devices `android.text.format.DateFormat`, so I would recommend you to change it.